test/downstream/observables_autodiff.jl

```diff
@@ -33,15 +33,15 @@ sol = solve(prob, Tsit5())
     gs, = gradient(sol) do sol
         sum(sol[sys.w])
     end
-    du_ = [0.0, 1.0, 1.0, 1.0]
+    du_ = [1.0, 1.0, 1.0, 0.0]
     du = [du_ for _ in sol.u]
     @test du == gs.u
 
     # Observable in a vector
     gs, = gradient(sol) do sol
         sum(sum.(sol[[sys.w, sys.x]]))
     end
-    du_ = [0.0, 1.0, 1.0, 2.0]
+    du_ = [1.0, 1.0, 2.0, 0.0]
     du = [du_ for _ in sol.u]
```

Comment:
```suggestion
    du = [du_ for _ in sol.u]
```

Just make this correct instead.